tests/unit/copying_manager_tests/copying_manager_new_test.py

```diff
@@ -301,10 +321,12 @@ def test_checkpoints(self):
         test_file.append_lines("Line7")
         test_file.append_lines("Line8")
 
-        for checkpoint_path in pathlib.Path(
-            self._env_builder.config.agent_data_path
-        ).glob("*checkpoints*.json"):
-            checkpoint_path.unlink()
+        # make sure that all worker session checkpoint files are consolidated and removed.
+        for worker_session in manager.worker_sessions:
```

Comment:
Can we please also add scenario where per worker active-checkpoints file is also created and assertion that this file is deleted as well?